specifyweb/frontend/js_src/lib/components/SchemaConfig/Fields.tsx

```diff
@@ -66,7 +66,15 @@ export function SchemaConfigFields({
 
         {relationships.length > 0 && (
           <optgroup label={schemaText.relationships()}>
-            <SchemaConfigFieldsList fields={relationships} />
+            <SchemaConfigFieldsList
+              fields={
+                table.name === 'CollectionObject'
+                  ? relationships.filter(
+                      ({ name }) => name !== 'collectionObjectType'
+                    )
```

Comment:
While it is not visible at the moment, I assume we'd want to do the same for `Collection -> collectionObjectType` once we have an SpLocaleContainerItem created in a migration?

Reply:
If we're gonna add those migrations in 7.9.7, then yes we should. However, I don't think we're planning to add those in this release.
I imagine this PR will be reverted sometime before releasing 7.9.8 so we just want to hide COType in all the places it is visible right now.

Reply:
Good catch on the mapper relationships!